src/RestaurantContainer.jsx

```diff
@@ -28,6 +42,14 @@ export default function RestaurantContainer({ restaurantId }) {
   return (
     <>
       <RestaurantDetail restaurant={restaurant} />
+      {accessToken ? (
+        <ReviewForm
+          fields={reviewFields}
+          onChange={handleChange}
+          onSubmit={handleSubmit}
+        />
+      ) : null}
```

Comment:
```suggestion
      {accessToken && (
        <ReviewForm
          fields={reviewFields}
          onChange={handleChange}
          onSubmit={handleSubmit}
        />
      )}
```
accessToken이 0이 되지 않는다면, 조금 더 간단하게 처리해줘도 좋겠네요 :)

(만약 비교하는 값이 0이 된다면 화면에 0이 노출됩니다. 이미 아실 수 있지만, false로 인식될 거라고 생각해서 실수 할 수 있는 부분이죠)

Reply:
오... 그 부분까지는 신경쓰지 못했어요....! 엄청 급하게 하느라..! 
accessToken이 0이 될 일은 없을 거 같으니까 && 로 처리하면 더 간단하네요!